src/app/shared/services/latestSearchesUtil.js

```diff
@@ -7,11 +7,20 @@ export default class LatestSearchesUtil {
         return this.latestSearches = JSON.parse(localStorage.getItem('latestSearches')) || [];
     };
 
-    addCity(city) {
-        if (!city || city.trim() === '' || this.latestSearches.includes(city)) {
+    addCity(rawCity) {
+        let city = rawCity.charAt(0).toUpperCase() + rawCity.slice(1).toLowerCase();
```

Comment:
Check if rawCity is not defined (if the user clicks on search button with empty input, an error appears on the console)

Reply:
also, why are you doing that at all?

Reply:
Woah, this one was strange, I already solved, didn't know why I did that, sorry!